DrawerKitDemo/DrawerKitDemo/PresentedViewController.swift

```diff
@@ -2,10 +2,25 @@ import UIKit
 import DrawerKit
 
 class PresentedViewController: UIViewController {
+    private var notificationToken: NotificationToken!
+
     @IBOutlet weak var presentedView: PresentedView!
     @IBAction func dismissButtonTapped() {
         dismiss(animated: true)
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.notificationToken = NotificationCenter.default.addObserver(name: DrawerNotification.drawerExteriorTapped.name) {
+            (notification: DrawerNotification, object: Any?) in
+            switch notification {
+            case .drawerExteriorTapped:
+                print("drawerExteriorTapped")
+            default:
+                break
+            }
+        }
+    }
```

Comment:
Aren't you missing remove the notification observer in this view controller?

Reply:
No. The view controller stores a notification token (an instance of a class) and, when the view controller goes out of scope, the token gets released, which invokes `deinit` on the token class, which removes the notification.